src/assets/stylesheet/modules/_buttons.scss

```diff
@@ -12,6 +12,7 @@ button, .btn {
   cursor: pointer;
   text-transform: uppercase;
   zoom: 1;
+  line-height: 20px;
```

Comment:
Rank C
Why did you adding line-height?

Reply:
Fix lỗi cho bên IE vì 1 button còn 1 cái span nên nó ko đều nhau